Move shim/xplat to shim_et/xplat, try 2

swolchok opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #8607

Looks like in order to get this right, we're going to have to update some internal configuration and originate this PR from internal repo. Supersedes #8583 / D69882927.

Differential Revision: [D69955361](https://our.internmc.facebook.com/intern/diff/D69955361/)

Commit:
Move shim/xplat to shim_et/xplat, try 2

Looks like in order to get this right, we're going to have to update some internal configuration and originate this PR from internal repo. Supersedes #8583 / D69882927.

Differential Revision: [D69955361](https://our.internmc.facebook.com/intern/diff/D69955361/)

[ghstack-poisoned]

larryliu0820 reviewed (approved):
Yeah I guess landing this without shipit will break Android build

swolchok committed:
Update on "Move shim/xplat to shim_et/xplat, try 2"


Looks like in order to get this right, we're going to have to update some internal configuration and originate this PR from internal repo. Supersedes #8583 / D69882927.

Differential Revision: [D69955361](https://our.internmc.facebook.com/intern/diff/D69955361/)

[ghstack-poisoned]

swolchok commented:
superseded by https://github.com/pytorch/executorch/pull/8647
